src/components/Books.js

```diff
@@ -0,0 +1,34 @@
+const Books = () => {
+  const books = [
+    {
+      title: 'First Book',
+      author: 'Anonimus',
+      id: 1,
+    },
+    {
+      title: 'Second Book',
+      author: 'Anonimus',
+      id: 2,
+    },
+  ];
+  return (
+    <div className="books-display">
+      <h2>This is a bookstore</h2>
+      <input type="text" placeholder="Add new book" />
+      <ul>
```

Comment:
- [ ] Kindly put this _**form control**_ along with a _**submit button**_ in a separate React component and render that component in the `Book` component. Creating of data should be in a separate component as per the separation of concerns concept. 👍